### ISSUE TITLE
Is the generated docs hosted somewhere?

### ISSUE THREAD

**Issue by yck011522** [dbt-ethz/compas_vol#6]

**Comment by yck011522** [comment]
If so, maybe add the link to the Git repro description or the README.md

**Comment by yck011522** [comment]
Oh, I found it. The link was there but its just not obvious it is the documentation.

https://dbt-ethz.github.io/compas_vol/

**Comment by worbit** [comment]
Good point, I will add it. The documentation can be found through compas core here: https://compas-dev.github.io/packages.html and there the link back to the repository will also be added.

**Comment by worbit** [comment]
not called packages anymore, but moved to extensions: https://compas-dev.github.io/extensions.html
